src/data-workspace/category-combo-table-body/category-combo-table-body.test.js

```diff
@@ -453,6 +464,7 @@ const metadata = {
 
 describe('<CategoryComboTableBody />', () => {
     useMetadata.mockReturnValue({ data: metadata })
+    useDataSetId.mockReturnValue(['dataSet1'])
```

Comment:
(FYI: edit was necessary because of thenew  call to the `useIsCompulsoryDataElementOperand` hook